pkg/config/config.go

```diff
@@ -48,10 +57,13 @@ type Config struct {
 
 	// LocalLinksStyle sets linking style to be applied. If empty, we assume default style.
 	LocalLinksStyle LocalLinksStyle `yaml:"localLinksStyle"`
+
+	// Validators set multiple composable validators for link checking.
+	Validators []ValidatorConfig `yaml:"validators"`
 }
 
 type TransformationConfig struct {
-	_glob glob.Glob
+	ParsedGlob glob.Glob
```

Comment:
```suggestion
	parsedGlob glob.Glob
```
If you make it public it will be parsable by yaml decoders (and other decoders )

Reply:
Sure! Also, these are both totally orthogonal things but as it is currently, a user would have to specify two different YAML files(if they choose not to go with just passing content) to make both transform and format work. Would that be okay? Then I can close this PR. 🙂

Reply:
Yes, why not? Those are two separate logics (:

Reply:
Two different commands even.

Reply:
Okay then! 🙂